### PR TITLE
Dev ticket15 add warehouse_name instead of warehouse_id 

### PR DESCRIPTION
**Pull request by Elizabeth-Sorescu**
Create an API on the back-end using Express and Express router to provide the front-end with a list of all Inventory items from all Warehouses. 

Note: The warehouse name for each inventory item must also be included in the response.

Data should be returned from your database using knex.

GET /api/inventories

Response returns 200 if successful

Response body example:

[
    {
      "id": 1,
      "warehouse_name": "Manhattan",
      "item_name": "Television",
      "description": "This 50\", 4K LED TV provides a crystal-clear picture and vivid colors.",
      "category": "Electronics",
      "status": "In Stock",
      "quantity": 500
    },
    {
      "id": 2,
      "warehouse_name": "Manhattan",
      "item_name": "Gym Bag",
      "description": "Made out of military-grade synthetic materials, this gym bag is highly durable, water resistant, and easy to clean.",
      "category": "Gear",
      "status": "Out of Stock",
      "quantity": 0
  },

### CODE REVIEW

**Comment by ryn-ngn** [comment]
@Elizabeth-Sorescu I changed the PR to merge into dev instead of main for this PR

File: controllers/inventory-controller.js [hunk 1]
```diff
@@ -3,7 +3,24 @@ const knex = require("knex")(require("../knexfile"));
...
+      container.id = item.id;
+      container.warehouse_name = dataWarehouse.find(
+        (w) => w.id === item.warehouse_id
+      ).warehouse_name;
+      container.item_name = item.item_name;
+      container.description = item.description;
+      container.category = item.category;
+      container.status = item.status;
+      container.quantity = item.quantity;
+
```
Comment (ryn-ngn) [inline controllers/inventory-controller.js:18]: Thanks Elizabeth, this works.

Another approach is to use object destructuring to clean up things a little bit from line 8 to 20:

const { id, warehouse_id, item_name, description, category, status, quantity } = item;
      const { warehouse_name } = dataWarehouse.find(w => w.id === warehouse_id);

      return {
        id,
        warehouse_name,
        item_name,
        description,
        category,
        status,
        quantity
      };
    })

**Comment by zahrafalak** [comment]
I wasn't able to verify that this works when I copied the code on my end. Were you able to get the warehouse name in the inventory list when you send a request to http://localhost:8080/api/inventories?

File: controllers/inventory-controller.js (hunk 1 above)
Comment (Elizabeth-Sorescu) [reply controllers/inventory-controller.js:18]: Hi Ryan, thanks for pointing this out. I have pushed the changes to this PR.

**Review by ryn-ngn — APPROVED** [review]

File: controllers/inventory-controller.js [hunk 2]
```diff
@@ -3,7 +3,35 @@ const knex = require("knex")(require("../knexfile"));
...
+        warehouse_id,
+        item_name,
+        description,
+        category,
+        status,
+        quantity,
+      } = item;
+
+      const { warehouse_name } = dataWarehouse.find(
+        (w) => w.id === warehouse_id
```
Comment (rjtbansal) [inline controllers/inventory-controller.js:20]: w is hard to tell. Feel free to always be descriptive.
` (warehouse) => warehouse.id === warehouse_id`

**Review by rjtbansal — COMMENTED** [review]
Looks good overall!

**Pull request merged**